src/build/firefox.manifest.json

```diff
@@ -14,7 +14,7 @@
 		"*://*.facebook.com/*",
 		"*://whotargets.me/*",
 		"*://*.whotargets.me/*",
-		"*://localhost:*/*"
```

Comment:
Sorry, I missed this, did it get merged like this? Oops, my bad.

It should be `"*://localhost/*"`. I wouldn't expect to see a `*` for a port range.

Reply:
changed